src/distilbert/distilbert_model.rs

```diff
@@ -682,30 +682,59 @@ impl DistilBertForTokenClassification {
     /// let config = DistilBertConfig::from_file(config_path);
     /// let distil_bert = DistilBertForTokenClassification::new(&p.root() / "distilbert", &config);
     /// ```
-    pub fn new<'p, P>(p: P, config: &DistilBertConfig) -> DistilBertForTokenClassification
-    where
-        P: Borrow<nn::Path<'p>>,
+    pub fn try_new<'p, P>(p: P, config: &DistilBertConfig) -> Result<DistilBertForTokenClassification, RustBertError>
+        where
```

Comment:
These edits do not appear when I run `rustfmt`, could you please see if you are using the latest binaries with default settings?

Reply:
Somehow intellij / cargo tricked me here :-/ ...